components/SubHeadline/SubHeadline.module.css

```
@@ -0,0 +1,25 @@
+.container {
+  height: 70px;
+  border-radius: 25px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 50% 50%;
+  color: var(--main-color);
+  padding-right: 25px;
+  padding-left: 25px;
+  align-content: center;
+}
+.subHeader {
+  align-self: end;
+}
+.sort {
+  color: var(--secundary-color);
+  font-size: 15px;
```

You should make this responsive wth `em` or `rem`
```suggestion
  font-size: 15rem;
```